pkg/volume/azure_file/azure_file.go

```diff
@@ -207,7 +209,12 @@ func (b *azureFileMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 	os.MkdirAll(dir, 0750)
-	source := fmt.Sprintf("//%s.file.core.windows.net/%s", accountName, b.shareName)
+
+	azure, err := getAzureCloud(b.plugin.host.GetCloudProvider())
```

Comment:
I like the idea of an override/fallback. I think that the default should be for it to be easy/automatic in the full azure environment, and then possible to override/extend in regional azure or external environments.